pytrip/vdx.py

```diff
@@ -1035,6 +1391,76 @@ def get_roi_type_name(type_id):
         """
         :returns: The type name of the ROI.
         """
+
+        # EXTERNAL
+        # external patient contour
+        #
+        # PTV
+        # Planning Target Volume (as defined in [ICRU Report 50])
+        #
+        # CTV
+        # Clinical Target Volume (as defined in [ICRU Report 50])
+        #
+        # GTV
+        # Gross Tumor Volume (as defined in [ICRU Report 50])
+        #
+        # TREATED_VOLUME
+        # Treated Volume (as defined in [ICRU Report 50])
+        #
+        # IRRAD_VOLUME
+        # Irradiated Volume (as defined in [ICRU Report 50])
+        #
+        # BOLUS
+        # patient bolus to be used for external beam therapy
+        #
+        # AVOIDANCE
+        # region in which dose is to be minimized
+        #
+        # ORGAN
+        # patient organ
+        #
+        # MARKER
+        # patient marker or marker on a localizer
+        #
+        # REGISTRATION
+        # registration ROI
+        #
+        # ISOCENTER
+        # treatment isocenter to be used for external beam therapy
+        #
+        # CONTRAST_AGENT
+        # volume into which a contrast agent has been injected
+        #
+        # CAVITY
+        # patient anatomical cavity
+        #
+        # BRACHY_CHANNEL
+        # brachytherapy channel
+        #
+        # BRACHY_ACCESSORY
+        # brachytherapy accessory device
+        #
+        # BRACHY_SRC_APP
+        # brachytherapy source applicator
+        #
+        # BRACHY_CHNL_SHLD
+        # brachytherapy channel shield
+        #
+        # SUPPORT
+        # external patient support device
+        #
+        # FIXATION
+        # external patient fixation or immobilization device
+        #
+        # DOSE_REGION
+        # ROI to be used as a dose reference
+        #
+        # CONTROL
+        # ROI to be used in control of dose optimization and calculation
+        #
+        # DOSE_MEASUREMENT
+        # ROI representing a dose measurement device, such as a chamber or TLD
+
```

Comment:
there are so many types in comment, but in code there are only 4 of them, why is it so?